missions/W2/M6/jupyter_server_config.py

```
@@ -0,0 +1,17 @@
+import os
+import sys
+
+sys.path.append('/app')
+
+
+c = get_config()
```

Ruff (0.8.2)

Undefined name `get_config`

(F821)